frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1056,7 +1063,7 @@ export const dashboardLogic = kea<dashboardLogicType>([
                     }
 
                     if (refreshedInsight.query_status) {
-                        pollForResults(refreshedInsight.query_status.id, false, methodOptions)
+                        await pollForResults(refreshedInsight.query_status.id, false, methodOptions)
                             .then(async () => {
```

Comment:
Do we still want `.then().catch() with `await`?

Reply:
Uh, right, that's maybe confusing then.